cw-orch-daemon/src/senders/multiple_sender.rs

```diff
@@ -0,0 +1,116 @@
+use crate::{DaemonBase, Wallet, INSTANTIATE_2_TYPE_URL};
+
+use crate::{error::DaemonError, tx_resp::CosmTxResponse};
+
+use cosmrs::proto::cosmwasm::wasm::v1::{MsgInstantiateContract, MsgStoreCode};
+use cosmrs::{AccountId, Any};
+use cosmwasm_std::Addr;
+use cw_orch_core::log::transaction_target;
+use prost::Name;
+
+use std::sync::{Arc, Mutex};
+
+use super::{base_sender::SenderOptions, sender_trait::SenderTrait};
+
+pub type MultiDaemon = DaemonBase<MultipleSender>;
```

Comment:
Can we rename `MultiDaemon` to `BatchDaemon` or `MultiTxDaemon` 

Same with `MultiSender` -> `BatchSender` / `MultiTxSender`?